src/main/java/model/process/ReversedSortProcess.java

```diff
@@ -0,0 +1,40 @@
+package model.process;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+import model.data.DataTable;
+import model.data.Table;
+import model.data.describer.RowValueDescriber;
+import model.data.value.DataValue;
+
+/**
+ * This class sorts a DataTable on whatever column you want it to sort on, but
+ * REVERSED.
+ * 
+ * @author Louis Gosschalk 10-06-2015
+ */
+public class ReversedSortProcess extends DataProcess {
```

Comment:
Zou je niet een Enum Order maken in SortProcess met ASCENDING en DESCENDING en dat dan in je constructor mee krijgen, dat scheelt best wel veel duplicate code.